vdom/create-element.js

```diff
@@ -26,6 +26,7 @@ function createElement(vnode, opts) {
         return null
     }
 
+    var tagName = vnode.tagName || "span"
```

Comment:
A span is the most minimal HTML element. Until recently it [directly implemented the `HTMLElement` interface](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span). HTML5 has changed this to [`HTMLSpanElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSpanElement), which is really nothing but an alias.

But it can just as well be a `div`, since it seems to match the conventions of `virtual-dom` better.